### PR TITLE
Travis-CI: Initial draft for CI builds.

### PR DESCRIPTION
**Pull request by sunweaver**

### CODE REVIEW

**Comment by sunweaver** [comment]
Ready for merging. Please review and merge, if ok.

File: CMakeLists.txt [hunk 1]
```diff
@@ -53,19 +53,19 @@ include_directories (${CMAKE_CURRENT_SOURCE_DIR}/src)
 include_directories (${CMAKE_CURRENT_BINARY_DIR}/src)
 
 # testing & coverage
-if (${enable_tests})
+if (${ENABLE_TESTS})
   set (GTEST_SOURCE_DIR /usr/src/gtest/src)
```
Comment (tari01) [inline CMakeLists.txt:57]: We need to have a test here:

```
if (EXISTS /usr/src/googletest/src)
    set (GTEST_SOURCE_DIR /usr/src/googletest/src)
else
    set (GTEST_SOURCE_DIR /usr/src/gtest/src)
endif ()
```

**Review by tari01 — CHANGES_REQUESTED** [review]
Fails when /gtest is /googletest

File: CMakeLists.txt (hunk 1 above)
Comment (sunweaver) [reply CMakeLists.txt]: Amended via extra commit. Please revisit.

**Review by tari01 — APPROVED** [review]

**Pull request merged**